po/fr.po

```diff
@@ -0,0 +1,382 @@
+# French translations for ConsoleKit2 package.
+# Copyright (C) 2020 The ConsoleKit2 Team
+# This file is distributed under the same license as the ConsoleKit2 package.
+# Jean Philippe EIMER <phil@jpmr.org>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: ConsoleKit2 1.2.2\n"
+"Report-Msgid-Bugs-To: https://github.com/ConsoleKit2/ConsoleKit2/issues\n"
+"POT-Creation-Date: 2017-12-13 15:40+0300\n"
+"PO-Revision-Date: 2020-12-30 16:10+0100\n"
+"Last-Translator: Jean Philippe EIMER <phil@jpmr.org>\n"
+"Language-Team: French <traduc@traduc.org>\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: src/ck-seat.c:253
+msgid "Another session was activated while waiting"
+msgstr "Une autre session a été activée pendant l'attente"
+
+#: src/ck-seat.c:279
+msgid "There is no Seat to activate"
+msgstr "Il n'y a pas de ticket à activer"
```

Comment:
Hi.
Are you sure "ticket" is the best translation for "Seat"?
Shouldn't it be "siège" or "poste de travail" or something alike?

Reply:
Hi,
The translation for 'Seat' is difficult in that case!
I chose "Ticket" because I believe it's better than "Siège" or "Place" or "Poste": for example, to book a seat for a concert, you get a ticket.
But I understand other people may have a different feeling.
I even wonder whether I should let Seat untranslated.

Reply:
systemd uses this as translation: `poste (seat)`
I think that is what makes the most sense.

Reply:
No Problem.
Feel free to modify this as you like.